contrib/babelfishpg_tds/src/include/tds_protocol.h

```diff
@@ -77,5 +77,7 @@ typedef struct
 } TdsRequestCtrlData;
 
 extern TdsRequestCtrlData *TdsRequestCtrl;
+extern bool resetTdsConnectionFlag;
+extern bool isRPCRequest;
```

Comment:
I dont think using extern variables is a good approach and passing them to the tsql extension looks like a hack, cant we pass a flag to the inline handler during our call from sp procedure call?